hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.html

```diff
@@ -865,12 +865,12 @@ <h4>Pension</h4>
                       ng-disabled="isDisabled"
                       ng-model="entity.pension.pension_type">
                       <ui-select-match
-                        placeholder="Start typing a name or email...">{{$select.selected.label}}</ui-select-match>
-                      <ui-select-choices  repeat="contact.id as contact in contacts.Pension_Provider | filter: $select.search"
-                        refresh="refreshContacts($select.search, 'Pension_Provider')"
+                        placeholder="Start typing a name...">{{$select.selected.label}}</ui-select-match>
+                      <ui-select-choices
+                        repeat="provider.value as provider in providers.pension_provider | filter: $select.search"
+                        refresh="loadProviderOptions(['hrjc_pension_provider'])"
                         refresh-delay="0">
-                        <div ng-bind="contact.label"></div>
-                        <small ng-bind="contact.description[0]"></small>
+                        <div ng-bind="provider.label"></div>
```

Comment:
Just noticed, is there any reason why we don't use `<div>{{provider.label}}</div>`? If there is no reason I would suggest to get rid of all `ng-bind` and replace with `{{ }}` in the whole document.